tools/internal_ci/linux/grpc_e2e_performance_v2.sh

```diff
@@ -88,6 +88,8 @@ buildConfigs() {
         -a ci_buildUrl="${CLOUD_LOGGING_URL}" \
         -a ci_jobName="${KOKORO_JOB_NAME}" \
         -a ci_gitCommit="${GRPC_GITREF}" \
+        -a ci_gitCommit_java="${GRPC_JAVA_GITREF}" \
```

Comment:
I would add go before java, so it is alphabetical.

You also need to add that to grpc_e2e_performance_gke.sh.